lib/imagery/list_gp.c

```diff
@@ -75,10 +78,45 @@ int I_list_group_simple(const struct Ref *ref, FILE * fd)
     int i;
 
     if (ref->nfiles <= 0)
-	return 0;
+        return 0;
 
     for (i = 0; i < ref->nfiles; i++)
-	fprintf(fd, "%s@%s\n", ref->file[i].name, ref->file[i].mapset);
+        fprintf(fd, "%s@%s\n", ref->file[i].name, ref->file[i].mapset);
 
     return 0;
 }
+
+/*!
+ * \brief Formats map name to fit in a 80 column layout
+ *
+ * Results in a map name in the "<map@mapset>" form.
+ * If necessary truncates relevant part(s) and denotes
+ * with ellipsis, e.g. "<verylongmapname...@mapset>".
+ *
+ * \param[out] buf formatted map name
+ * \param name map name
+ * \param mapset mapset name
+ */
+static void name_fitted(char *buf, const char *name, const char *mapset)
```

Comment:
Why not making the function shared? If you don't think this should be part of API, make it double underscore `I__...`. If it would be a one liner, but with the current code...

Reply:
The function is general enough for lib API. The question is whether it may be of use elsewhere. Anyone know?

Reply:
Just updated with the function now as part of lib/imagery API.

Reply:
Moved this part of the PR to #1818.